test-lib.sh

```diff
@@ -456,13 +456,26 @@ ct_s2i_build_as_df()
     local user_id=
     local df_name=
     local tmpdir=
+    local incremental=false
     # Use /tmp to not pollute cwd
     tmpdir=$(mktemp -d)
     df_name=$(mktemp -p "$tmpdir" Dockerfile.XXXX)
     pushd "$tmpdir"
     # Check if the image is available locally and try to pull it if it is not
     docker images "$src_image" &>/dev/null || echo "$s2i_args" | grep -q "pull-policy=never" || docker pull "$src_image"
     user_id=$(docker inspect -f "{{.Config.User}}" "$src_image")
+    echo "$s2i_args" | grep -q "\-\-incremental" && incremental=true
+    if [ $incremental == true ]; then
+        inc_tmp=$(mktemp -d --tmpdir incremental.XXXX)
+        setfacl -m "u:$user_id:rwx" "$inc_tmp"
```

Comment:
I'm just not sure that `$user_id` really contains numerical value, so I'd prefer a check for this.

Reply:
That is a good point. I tried it out and it turns out the config value is anything you give the `USER ...` command in the Dockerfile which can include username instead of a numerical value. It can also be empty (if no `USER ...` command is used) or a username of a non-existing user.

Will make this more bullet-proof.